### ISSUE TITLE
play file .raw 

### ISSUE THREAD

**Issue by HuongNV96** [xiph/rnnoise#104]
Hi all

When I downloaded data in "https://people.xiph.org/~jm/demo/rnnoise/rnnoise_contributions.tar.gz" link , Can I play them as music (.mp3 file ) ?
If it can , Could you help me to show proper command or software ?

Very thank you

**Comment by roy-bankesh** [comment]
No, All are raw file present there you have to convert the raw file into other format. You can use this link https://github.com/smallmuou/wavutils  to get a tool to convert. This one is orking fine for me.

**Comment by Zadagu** [comment]
You can use Audacity to listen these files. Use File -> import -> raw audio. Set sample rate to 48000 and sample size to signed 16bit pcm.

**Comment by slavaGanzin** [comment]
I use sox:
```play --type raw --rate 8k --bits 16 --channels 1 --encoding signed in.wav```

And you can use sox for convertion, just change `play` to `sox` like:
```sox --type raw --rate 8k --bits 16 --channels 1 --encoding signed in.wav out.mp3```
